### PR TITLE
[unit-tests] added unit tests for OcSearchBar component

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
- Added unit tests for `OcSearchBar` component

## Related Issue

- Part of #1366 

## How Has This Been Tested?

- 💇🏼‍♂️ 
- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ x Tests

## Checklist:

- [ ] Code changes
- [ x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation added/updated

## Open tasks:

- [ ] ...

### CODE REVIEW

File: src/components/OcSearchBar.spec.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        expect(spinnerStub.exists()).toBeTruthy()
+        expect(spinnerStub.attributes("style")).toBe("display: none;")
+      })
+      it("should enable search input", () => {
+        const searchInput = wrapper.find(selectors.searchInput)
+        expect(searchInput.attributes("disabled")).toBe(undefined)
+      })
+      it("should enable search button", () => {
+        const searchInput = wrapper.find(selectors.searchInput)
+        expect(searchInput.attributes("disabled")).toBe(undefined)
```
Comment (individual-it) [inline src/components/OcSearchBar.spec.js:107]: that looks like the same test

**Review by individual-it — CHANGES_REQUESTED** [review]

File: src/components/OcSearchBar.spec.js (hunk 1 above)
Comment (kiranparajuli589) [reply src/components/OcSearchBar.spec.js]: ahh..should be checking for the button.
fixed

**Review by individual-it — COMMENTED** [review]
tests look good
@pascalwengerter are you happy with the code changes?

**Review by pascalwengerter — APPROVED** [review]
LGTM!

**Review by individual-it — APPROVED** [review]

**Pull request merged**